compose/hmi.yml

```diff
@@ -42,6 +42,23 @@ services:
     networks:
       - hmc-network
 
+  hmc-cft-hearing-service:
+    image: "${HMC_CFT_HEARING_SERVICE_USE_LOCAL-hmctspublic.azurecr.io/}hmc/cft-hearing-service:${HMC_CFT_HEARING_SERVICE_TAG:-latest}"
```

Comment:
We need to add the new service into the `hmc` file to allow the `./hmc set ....` commands etc. to work

Reply:
added to the hmc file